src/components/SRNEntry.vue

```diff
@@ -130,13 +133,30 @@ export default {
         this.validateCount,
         this.isSingleEntryOnly,
         this.redirectID,
-        this.doesUserExist
+        this.isUserValid,
+        this.purpose,
+        this.purposeParams,
+        this.redirectTo
       );
 
       userIsValidated.then((result) => {
-        this.doesUserExist = result.doesUserExist;
+        this.isUserValid = result.isUserValid;
```

Comment:
Do we need these to be global variables? Looks like you can simply use the result directly to construct the message to SQS?

Reply:
I think yes, as these variables are being used across the code for conditional rendering of html